### PR TITLE
ENH: --ignore-docs and exit code.

### LINKED ISSUE #15: requirements-compare add option to skip docs and exit code
## Feature Request
- Add option to skip the check of docs requirements since a `docs` entry at meta.yaml will be a violation at conda-forge. If we are okay with keeping this extra at the PCDS side I am fine with it since the extra field doesn't seem to cause an issue with `conda-build`.

- Make requirements-compare exist with 1 instead of 0 so we can use it directly at CI.

## Current Behavior
requirements-compare exits with 0 even when files have inconsistency.

### PR DESCRIPTION
**Pull request by hhslepicka**
## Description
- Add `--ignore-docs`
- Set exit code to 1 in case of differences at `requirements-compare`.

## Motivation and Context
Closes #15 

## How Has This Been Tested?
Locally

## Where Has This Been Documented?
N/A

### CODE REVIEW

File: pcdsutils/requirements.py [hunk 1]
```diff
@@ -213,3 +218,4 @@ def _compare_requirements(args=None):
                     else:
                         logger.info('- %s', item)
                 logger.info('')
+    return 1 if has_diff else 0
```
Comment (ZryletTC) [inline pcdsutils/requirements.py:221]:
```suggestion
    return 1 if has_diff else 0
    
```

Comment (hhslepicka) [reply pcdsutils/requirements.py]: Good catch.. I always forget that... 😞

Comment (ZryletTC) [reply pcdsutils/requirements.py]: Very easy to forget. I would forget it all the time if my editor didn't automatically check.

Comment (ZryletTC) [reply pcdsutils/requirements.py]: Oops, I guess there was some whitespace in there.

Comment (hhslepicka) [reply pcdsutils/requirements.py]: I just pushed a fix...

**Pull request merged**